pom.xml

```diff
@@ -19,11 +19,15 @@
     Additionally, the plug-in provides health reporting and build stability based on these combined results.
   </description>
 
+  <properties>
+    <workflow.version>1.7</workflow.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.jvnet.hudson.plugins</groupId>
       <artifactId>analysis-core</artifactId>
-      <version>1.73-SNAPSHOT</version>
+      <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
Right, I think this was just an artifact of how the upstream PR got `deploy`ed. For whatever reason `1.73-beta-SNAPSHOT` was deployed, even though there is no plan to release `1.73-beta` (that I know of).